### DISCUSSION TITLE
我续费升级到了专业版本，但是大部分时间都没办法访问youtube,google等网站，偶尔可以访问。邮箱314507871@qq.com    麻烦处理一下。

### DISCUSSION THREAD

**Discussion by PiratesFirst** [getlantern/lantern-forum-cn#281] [Q&A]
![image](https://github.com/getlantern/lantern-forum-cn/assets/4718078/920ccf95-29cf-4c67-b27e-489e0ba868b6)

**Comment by MaxMKMax** [comment]
@PiratesFirst 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

我们将非常感谢您。

**Reply by PiratesFirst** [reply]
号码是：70800599

![Uploading 微信图片_20240328135353.png…]()

**Reply by MaxMKMax** [reply]
@PiratesFirst 好的. 超级谢谢您! 我会开个ticket来反馈这个问题. 之后科技团队会进行修复了.

**Reply by PiratesFirst** [reply]
@Mike777Khoo   不行啊  ，今天直接闪退了。     重新安装最新版本都闪退
